### PR TITLE
Max attachments

### PR DESCRIPTION
**Pull request by hannah-bulmer**
The integration tests are the tests that are failing because they cannot connect to SFTP server. It is not related to this PR, as they also fail on the master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@
 
 This component creates a connection to an SFTP server to read and upload files.
 
+The optional environment variable `MAX_FILE_SIZE` should be set in settings to provide the maximum file size that can be uploaded in **bytes**. The default value for `MAX_FILE_SIZE` is 3.5GB.
```
Comment (jhorbulyk) [inline README.md]: Max size should be 100 MB.  It's not clear if that is `100*1000^2` or `100*1024^2` bytes.

File: lib/attachments.js [hunk 2]
```diff
@@ -5,8 +5,13 @@ const { Transform, Readable } = require('stream');
 const path = require('path');
 const { unixTimeToIsoDate } = require('../lib/utils/utils');
 
+const MAX_FILE_SIZE = process.env.MAX_FILE_SIZE || 3500000;
```
Comment (jhorbulyk) [inline lib/attachments.js]: This is 3.5 MB not 3.5 GB

**Comment by jhorbulyk** [comment]
Solves #51

File: README.md (hunk 1 above)
Comment (hannah-bulmer) [reply README.md]: It is the latter; file sizes are returned in bytes

File: README.md [hunk 3]
```diff
@@ -24,6 +24,8 @@
 
 This component creates a connection to an SFTP server to read and upload files.
 
+The optional environment variable `MAX_FILE_SIZE` should be set in settings to provide the maximum file size that can be uploaded in **megabytes (mb)**. The default value for `MAX_FILE_SIZE` is 100MB.
```
Comment (shulkaolka) [inline README.md]: @hannah-bulmer why did you decide to use 100MB? Did you test such attachment size on the platform?
@stas-fomenko as I know currently it is something like 10MB?

Comment (jhorbulyk) [reply README.md]: Tigran updated the platform documentation from 10MB to 100MB a couple of days ago: https://github.com/elasticio/elasticio.github.io/pull/438 We should double check to make sure that 100 MB works correctly.

Comment (stas-fomenko) [reply README.md]: `SFTP: Download file by name` cannot get ~97mb file with `EIO_REQUIRED_RAM_MB` = 4000, 
Component throws an error - `Error: Component did not send any data during timeout.`

Actually I thought that limit is 10mb - in this case test cases passes

10+MB files usually uploaded incorrectly even with additional memo, so if actual limit is 100MB then SFTP component has a bug with handling big (<100 mb) files

**Pull request merged**